src/main/java/com/imsweb/datagenerator/naaccr/rule/tumor/StagingInputRule.java

```diff
@@ -36,7 +36,7 @@ public List<String> getRequiredProperties() {
     }
 
     @Override
-    @SuppressWarnings("unchecked")
+    @SuppressWarnings({"unchecked", "java:S1905"})
```

Comment:
I am not against suppressing some errors, but when you do, can you please put a short comment describing what you are suppressing. If someone works on this code, they should know whether they should remove the suppression or not, but it's really not obvious with just those "java:Sxxxx" values. No need to write too much, but something like "// suppressed if statement simplification because I find the current code prettier", or anything you want, as long as it summarizes what you are suppressing.